src/releng_docs/deploying.rst

```diff
@@ -1,146 +1,244 @@
 
-Deploying of new services - at this time - does not handle creation/removal of
-S3 buckets / Heroku applications. Those need to be created beforehand.
 
-Manually deploying
-------------------
+Release Schedule
+----------------
 
-To deploy a single service manually, you need to run:
+Releases happen on a **weekly** basis, excluding `Firefox release week`_. `Next
+release`_ is announced in github issue tracker.
 
-.. code-block:: bash
 
-    % make deploy-staging APP=releng_clobberer ...
-        or
-    % make deploy-production APP=releng_clobberer ...
+Any **out-of-schedule** release can be requested by contacting :ref:`release
+manager <release-managers>` via irc or `creating a ticket on github`_.
 
-Above commands are missing many secrets which you need to provide as variables
-to make commands above. You can check how this is done on taskcluster via
-``.taskcluster.sh`` script.
+.. _`creating a ticket on github`: https://github.com/mozilla-releng/services/issues/new
+.. _`Next release`: https://github.com/mozilla-releng/services/issues?q=is%3Aopen+is%3Apr+label%3A%220.kind%3A+release%22
+.. _`Firefox release week`: https://wiki.mozilla.org/RapidRelease/Calendar
 
-Secrets are retrived from `taskcluster secrets services`_ for each branch
-separatly.
 
-- staging => `repo:github.com/mozilla-releng/services:branch:staging`_
-- production => `repo:github.com/mozilla-releng/services:branch:production`_
+Release Protocol
+----------------
 
+This protocal is followed by the :ref:`release manager <release-managers>` of ``mozilla-releng/services``.
 
-URL naming scheme
------------------
+#. If not already there, create a PR from ``master`` to ``production`` branch.
+   (`Example of release PR`_).
+
+   Please make sure that release notes are collected in description of the PR.
+   Encurage developers adding release notes as their PR gets merged into
+   ``master``.
+
+#. A day prior to the scheduled release send an email to all :ref:`service
+   owners <service-owners>`
+
+   This email should announce and remind everybody that release is going to
+   happen to avoid surprises.
+   
+   Template for email::
+
+       SUBJECT:
+
+           mozilla-releng/services release v<VERSION> is going to
+           happen on <RELEASE_DATETIME>
+
+       BODY:
+
+           Hi,
+
+           Next planned mozilla-releng/services relase is going to
+           happen tomorrow. We encurage everybody that contributed
+           to this release to join "#shipit" channel where release
+           is going to be coordinated. 
+
+               YYYY-MM-DD MM:HH CET
+               YYYY-MM-DD MM:HH PST
+
+           Please follow the link bellow for more details.
+
+               <LINK-TO-RELEASE-PR>
+
+           For any question please contact
+
+               <CURRENT-RELEASE-MANAGER>
+
+          
+#. Release starts by :ref:`release manager <release-managers>` loging all the
+   steps into ``#shipit`` channel and coordinating with others.
```

Comment:
I don't think #shipit is too noisy, and this will only happen once a week for half an hour. I think #shipit is a good place to start this (for now) and we can move this to another channel later.